bbmri/modules/directory-sync-compose.yml

```diff
@@ -6,7 +6,11 @@ services:
     environment:
       DS_DIRECTORY_URL: ${DS_DIRECTORY_URL}
       DS_DIRECTORY_USER_NAME: ${DS_DIRECTORY_USER_NAME}
-      DS_DIRECTORY_PASS_CODE: ${DS_DIRECTORY_PASS_CODE}
+      DS_DIRECTORY_USER_PASS: ${DS_DIRECTORY_USER_PASS}
       DS_TIMER_CRON: ${DS_TIMER_CRON}
       # It is recommended to check the enabling of this flag with your local data protection group
       DS_DIRECTORY_ALLOW_STAR_MODEL: ${DS_DIRECTORY_ALLOW_STAR_MODEL}
+      DS_DIRECTORY_MOCK: ${DS_DIRECTORY_MOCK}
+      DS_DIRECTORY_DEFAULT_COLLECTION_ID: ${DS_DIRECTORY_DEFAULT_COLLECTION_ID}
+      DS_DIRECTORY_COUNTRY: ${DS_DIRECTORY_COUNTRY}
+    command: sh -c "sleep 90 && java -jar directory_sync_service.jar" # Wait for Blaze before start
```

Comment:
Blaze might not be up after 90 secs, or gone down in the meantime. All our apps should be resilient towards failures of other apps. Can you error-handle this in the service itself?

Also, since you obviously rely on Blaze, consider adding a dependency in Docker (not a guarantee but doesn't hurt)

Reply:
The service will retry if Blaze is not there.

I added a "depends_on" in my last git push.